QEfficient/__init__.py

```diff
@@ -9,6 +9,7 @@
 from QEfficient.compile.compile_helper import compile
 from QEfficient.exporter.export_hf_to_cloud_ai_100 import qualcomm_efficient_converter
 from QEfficient.generation.text_generation_inference import cloud_ai_100_exec_kv
+from QEfficient.peft import QEffAutoPeftModelForCausalLM
```

Comment:
`Couple of points:
`
- The caching changes here are specific to PEFT or apply to overall repo? If not, then are we following two paths currently to save PEFT models and mainline models separately?
- Do we have details in the ReadMe on usage of this class? like passing adapters along with the model name to initiate this?

Reply:
1. Caching is specific to PEFT for now. I'm creating a separate PR to have this available to `QEffAutoModelForCausalLM` class next.
2. I have added the sample usage in the docstring, and it will be added in the github.io pages when merged. See docs/source/hl_api.md file for the new section.